apps/api/src/rest/auth.ts

```
@@ -0,0 +1,311 @@
+import dayjs, { Dayjs } from 'dayjs';
+import { and, eq, gt } from 'drizzle-orm';
+import escape from 'escape-string-regexp';
```

Biome (1.9.4)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)